packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -1338,33 +1208,69 @@ For more information about configuring an app for OpenID Connect, including grou
 * The full set of claims for the requested scopes is available via the [/oauth2/v1/userinfo](#userinfo) endpoint. Call this endpoint using the access token.
 
 ### Refresh Token
-
 Refresh tokens are opaque. More information about using them can be found in the [Authentication Guide](/authentication-guide/tokens/refreshing-tokens).
 
 ## Token Authentication Methods
```

Comment:
@danielzeller-okta Do you know why we call this "token authentication"? We're talking about authenticating endpoints (for more than just the /token endpoint) here, and methods like `client_secret_basic` are definitely not token authentication.

Reply:
After digging a little more I'm guessing it's historical - at one time, it just referred to the /token endpoint?

Reply:
@nbarbettini mostly because the actual field which determines what authentication mechanism the client uses is named `token_endpoint_auth_method` in the Dynamic Client Registration spec. Naming it "Client Authentication Methods" might make it more clear?

Reply:
@nbarbettini and @danielzeller-okta 
I actually debated with myself over this very topic. In the end, I left it and planned to discuss with you all. So, here we are. :) I think Client Authentication Methods is more clear and what I originally wanted to change it to. Sound good?

Reply:
DONE